### PR TITLE
Disable metrics by default

### PR DESCRIPTION
**Pull request by majestrate**

### CODE REVIEW

File: llarp/config/config.cpp [hunk 1]
```diff
@@ -266,13 +266,21 @@ namespace llarp
   void
   MetricsConfig::fromSection(string_view key, string_view val)
   {
-    if(key == "disable-metrics")
+    if(key == "enable-metrics")
     {
-      disableMetrics = true;
+      disableMetrics = IsFalseValue(val);
```
Comment (michael-loki) [inline llarp/config/config.cpp]: I think if we’re doing it like this, we should rename the variable to `enableMetrics`

Comment (majestrate) [reply llarp/config/config.cpp]: i added an option for that too

Comment (michael-loki) [reply llarp/config/config.cpp]: I mean, if metrics are disabled by default, we should make the variable “enableMetrics”, with a default value of false. 

Positive bools are easier to understand imo.

Comment (majestrate) [reply llarp/config/config.cpp]: i agree

**Pull request closed without merging**